src/actions/periodicMessage.ts

```diff
@@ -60,24 +60,10 @@ export const periodicMessage = async ({
   }
 
   if (shouldSend) {
-    if (guild.boundMessageId) {
-      const existingMessage = await channel.messages.fetch(
-        guild.boundMessageId
-      );
-
-      if (existingMessage) {
-        await existingMessage.delete();
-      }
-    }
-
-    const msg = await getEventsDetails({
+    await getEventsDetails({
       guildId: guildId,
       channel: channel,
     });
-    if (msg) {
-      guild.boundMessageId = msg.id;
-      guildRepository.save(guild);
-    }
```

Comment:
We now do this in the report command. So any time we list the details, it rebind that post as the most recent message